swift-ticketing/src/swift_ticketing/db/query.clj

```diff
@@ -0,0 +1,9 @@
+(ns swift-ticketing.db.query
+  (:require [next.jdbc :as jdbc]
+            [next.jdbc.result-set :as rs]))
+
+(defn run-query
+  ([db-spec query-fn args opts]
+   (jdbc/execute! db-spec (apply query-fn args) opts))
```

Comment:
We only seem to be using the 3 argument variant of this function. You can remove the 4 argument version for now.